Makefile

```
@@ -315,7 +315,7 @@ ifneq ($(findstring ocrd_pc_segmentation, $(OCRD_MODULES)),)
 OCRD_EXECUTABLES += $(OCRD_PC_SEGMENTATION)
 OCRD_PC_SEGMENTATION := $(BIN)/ocrd-pc-segmentation
 $(OCRD_PC_SEGMENTATION): ocrd_pc_segmentation
-	$(pip_install)
+	. $(ACTIVATE_VENV) && cd $< && $(MAKE) install
```

How about best of both worlds?

```suggestion
	. $(ACTIVATE_VENV) && $(MAKE) -C $< deps
	$(pip_install)
```

That way we get all dependencies as needed, but the package itself will be installed such that timestamps are guaranteed to be updated (and in `-e` mode if requested).